### PR TITLE
Profile data

### PR DESCRIPTION
**Pull request by Jeroen777**
- I've added some data to fill the profile (countries & preferences)
- I had to change some small things in the profile model so the data for the preferences wont delete the current profile data.
- And some small css changes

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -370,7 +370,9 @@ function showProfile(req, res) {
 		} else {
 			res.render('pages/profile', {
 				title: 'Profile',
-				profileData: result.profileData
+        profileData: result.profileData,
+        countryView: result.countries,
+        preferences: result.preferences,
```
Comment (daoneandonly) [inline app.js:375]: Let's indent this a little neater. If this only happens on github you can ignore this.

File: src/scss/pages/profile.scss [hunk 2]
```diff
@@ -11,4 +11,23 @@
     width: 200px;
 }
 
+}
+
+.profileCountries, .profilePreferences {
+    font-size: 0.8em;
+    text-align: center;
```
Comment (daoneandonly) [inline src/scss/pages/profile.scss]: I'd heavily argue against center aligning text. Maybe center align the element and left align the text for better readability.

**Review by daoneandonly — CHANGES_REQUESTED** [review]
Some minor changes, and then this is ready to go. Good work 💪

File: app.js (hunk 1 above)
Comment (Jeroen777) [reply app.js:375]: Yes I think this is just a small github thing. In my app.js the changed data is aligned like the removed line on 373.

**Review by daoneandonly — APPROVED** [review]

**Pull request merged**